docs/tutorial/colocation.md

```
@@ -199,14 +199,14 @@ $ kubectl exec shared-large-pod-2 -it -- stress -c 40
 ```
 status:
     resourceAllocatable:
-        katalyst.kubewharf.io/reclaimed_millicpu: 4k
+        resource.katalyst.kubewharf.io/reclaimed_millicpu: 4k
     resourceCapacity:
-        katalyst.kubewharf.io/reclaimed_millicpu: 4k
+        resource.katalyst.kubewharf.io/reclaimed_millicpu: 4k
 ```
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)